### DISCUSSION TITLE
curl error

### DISCUSSION THREAD

**Discussion by ThomLopp** [davidgohel/flextable#526] [Q&A] [answered]
I have a computer that for security reasons can't connect to the internet via curl. 
Is there a way to contour avail_gfonts() when printing html?

For example :
my_data <- data.frame(
  name = c("Alice","Alice", "Bob", "Charlie"),
  age = c(25, 29, 30, 35),
  city = c("New York", "London","Paris", "Tokyo")
)
test = flextable(my_data) 
print(test, "html")

I get error
Error in curl::curl_fetch_memory(x$url$url, handle = x$url$handle) : 
  Timeout was reached: [gwfh.mranftl.com] Failed to connect to gwfh.mranftl.com port XXX after XXXX ms: Timed out

**Comment by davidgohel** [comment, marked as answer]
I know sorry, it's happening here https://github.com/davidgohel/flextable/issues/525

If you really don't have internet, the fix should work (but here is a catch22, if you don't have web access, you cant install the github version...)
